homeassistant/components/whirlpool/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Whirlpool Sixth Sense",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
```

Comment:
Host is not a key used according to the config flow.

Reply:
oops